homeassistant/components/here_travel_time/sensor.py

```diff
@@ -216,55 +225,32 @@ def _are_valid_client_credentials(here_client: herepy.RoutingApi) -> bool:
     return True
 
 
-class HERETravelTimeSensor(SensorEntity):
+class HERETravelTimeSensor(SensorEntity, CoordinatorEntity):
     """Representation of a HERE travel time sensor."""
 
     def __init__(
         self,
         name: str,
-        origin: str,
-        destination: str,
-        origin_entity_id: str,
-        destination_entity_id: str,
-        here_data: HERETravelTimeData,
+        traffic_mode: str,
+        coordinator: HereTravelTimeDataUpdateCoordinator,
     ) -> None:
         """Initialize the sensor."""
         self._name = name
-        self._origin_entity_id = origin_entity_id
-        self._destination_entity_id = destination_entity_id
-        self._here_data = here_data
-        self._unit_of_measurement = TIME_MINUTES
```

Comment:
Set `_attr_unit_of_measurement` here and remove the`native_unit_of_measurement` property below.

Reply:
Should be `_attr_native_unit_of_measurement`

Reply:
Thanks!